plugins/broker-kafka/src/index.ts

```diff
@@ -345,6 +355,61 @@ class KafkaPlugin implements AmplicationPlugin {
 
     return modules;
   }
+
+  beforeCreateConnectMicroservices(
+    context: DsgContext,
+    eventParams: CreateConnectMicroservicesParams
+  ): CreateConnectMicroservicesParams {
+    const { template } = eventParams;
+
+    const generateKafkaClientOptionsImport = importNames(
+      [builders.identifier("generateKafkaClientOptions")],
+      "./kafka/generateKafkaClientOptions"
+    );
+
+    addImports(
+      template,
+      [generateKafkaClientOptionsImport].filter(
+        (x) => x //remove nulls and undefined
+      ) as namedTypes.ImportDeclaration[]
```

Comment:
```suggestion
      ) as namedTypes.ImportDeclaration[]
```

Why do we need to cast this?

Reply:
@overbit This is a copy-paste function :)  I can check it without the casting.